pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -1948,6 +1948,25 @@ def get_voronoi_volume(self):
         return self._analyse.pyscal_voronoi_volume()
     get_voronoi_volume.__doc__ = Analyse.pyscal_voronoi_volume.__doc__
 
+    def is_skewed(self, tolerance=1.0e-8):
```

Comment:
This should work, but isn't it more straightforward to check the dot product between cell vectors?  Your reasoning is documented though, so I don't mind too much.

Reply:
Uh that's a good point. I don't have a strong opinion about it. The only one thing is that I want to insist on making `tolerance` a dimensionless value, in which case the computational cost is probably very much the same. If other people prefer dot product I can also change it.